### DISCUSSION TITLE
Sound doesn't mute when it's 0%

### DISCUSSION THREAD

**Discussion by kahyalar** [MonitorControl/MonitorControl#1692] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

When I pull back that slider to 0%, in the monitor it shows it's muted but I am still getting sound from my speakers. My speakers(AudioEngine A2+ are attached to monitor via 3.5mm jack.)

### Expected behavior

When I pull down volume to 0%, I expect it's muted.

### Environment Information (please complete the following information)

```markdown
- macOS version: 15.1.1 Sequoia
- Mac model: Mac Studio M1 Max (2022)
- MonitorControl version: v.4.3.3 - Build 7123
- Monitor(s): LG 27UP850-W
- Monitor Cable(s)/Connection(s): Mac Studio M1 -> USB-C -> LG 27UP850-W
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment, marked as answer]
I recommend using BetterDisplay and configure mute support in a way that works with your system:

https://github.com/waydabber/BetterDisplay/issues/1248
